### DISCUSSION TITLE
Bluetooth Behavior

### DISCUSSION THREAD

**Discussion by ChadwickTheCrab** [sle118/squeezelite-esp32#434] [Q&A]
Hey! I have an esp32 device that is always on and an outdoor bluetooth speaker that I turn on when outside. It seems I need to power on the speaker and reboot the esp32 to get the Bluetooth connected. Is there a setting to have the esp32 continuously look for its target and connect? The only setting I saw that might be related is "Output device close timeout" which by default was set at 30. Should this be blank or another value? Thanks!

**Comment by sle118** [comment]
Output to Bluetooth is something of a hack that I introduced early in the beginning of the project while I was waiting for my DAC board to arrive. It is what it is, a mashup of pour early porting efforts and some Bluetooth demo from espressif.

You could probably hack your way through. If you succeed, I will be glad to review a pull request
